### ISSUE TITLE
Search RPS decreased comparing with 0.10.*

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#3968]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
![newplot](https://user-images.githubusercontent.com/56623710/95750404-a9b66a80-0ccf-11eb-8215-652e6d7b4b6f.png)

**Steps/Code to reproduce behavior**
Run on server: eros
Test tool: `locust`, using test suite: `suites/011_locust_search.yaml`, in `Test/milvus_benchmark` repo
Single connection sharing between 100 clients, 20 clients hatched per second
Collection num: 1
Details:
```
based on dataset: sift-1m-1024(index_file_size)/2000000(row_limit)_128d_l2
index_type: ivf_sq8
search_params:
   top_k: 10
   nrpobe: 16
   nq: 1
```
**Expected behavior**

**Method of installation**
- [x] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
      server:
        cache_config.cpu_cache_capacity: 8GB
        cache_config.insert_buffer_size: 2GB
        engine_config.use_blas_threshold: 1100
        engine_config.gpu_search_threshold: 1
        gpu_resource_config.enable: false
        gpu_resource_config.cache_capacity: 4GB
        gpu_resource_config.search_resources:
          - gpu0
          - gpu1
        gpu_resource_config.build_index_resources:
          - gpu0
          - gpu1
        wal_enable: true
      collection_name: sift_1m_2000000_128_l2_2
      ni_per: 50000
      build_index: true
      index_type: ivf_sq8
      index_param:
        nlist: 16384
      task:
        type: query
        connection_num: 1
        clients_num: 100
        hatch_rate: 2
        during_time: 5m
        params:
          top_k: 10
          nq: 1```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

**Comment by shengjun1985** [comment]
Search combination has not been implemented in 0.11.0.

**Comment by del-zhenwu** [comment]
nq: 1, the CPU usage:
```

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                                                                
 8412 root      20   0 47.751g 1.718g 745760 S  1600  0.3   1195:50 milvus_server  
```
